### ISSUE TITLE
Problem: integration with Enduro is not available

### ISSUE THREAD

**Issue by sevein** [artefactual-labs/a3m#40]
Enduro is currently able to communicate with pre-provisioned Archivematica pipelines, but working with a3m is a planned feature too.

a3m bundles an Enduro activity worker that can be easily deployed using our Docker image. Configuration can be passed as environment strings or request-scoped attributes. The main parameters are Cadence server address, domain name and task list name. The following is an example of a3m deployed as a Kubernetes Deployment with eight replicas:

```yaml
---
apiVersion: v1
kind: Namespace
metadata:
  name: a3m
---
apiVersion: v1
kind: Secret
metadata:
  name: a3m-docker-registry-key
  namespace: a3m
data:
  .dockerconfigjson: ...
type: kubernetes.io/dockerconfigjson
---
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: a3m
  name: a3m
  namespace: a3m
spec:
  replicas: 8
  selector:
    matchLabels:
      app: a3m
  template:
    metadata:
      labels:
        app: a3m
    spec:
      imagePullSecrets:
      - name: a3m-docker-registry-key
      containers:
      - name: a3m
        image: docker.pkg.github.com/artefactual-labs/a3m/a3m:main
        imagePullPolicy: Always
        env:
        - name: "A3M_CADENCE_SERVER"
          value: "cadence-server:12345"
        - name: "A3M_CADENCE_DOMAIN"
          value: "a3m"
        - name: "A3M_CADENCE_TASK_LIST"
          value: "a3m"
```

Enduro needs to learn how to utilize a3m:
* Dispatch activities to the `a3m` task list
* Encode activity request (including name and URL of the transfer)
* Give access to transfer, e.g. via S3 pre-signed URLs

**Comment by sevein** [comment]
Activity worker implemented here: e9a3398a81abe5ca4cb58740c74a168dc885b5a0. Not tested yet.

**Comment by sevein** [comment]
Not part of this repo.
